src/Config/Auth.php

```diff
@@ -42,6 +42,16 @@ class Auth extends BaseConfig
         'magic-link-email'            => '\CodeIgniter\Shield\Views\Email\magic_link_email',
     ];
 
+    /**
+     * --------------------------------------------------------------------
+     * Customize the DB group used for each model
+     * --------------------------------------------------------------------
+     * if no database connection instance is passed to the constructor,
+     * it will automatically connect to the default database group,
+     * as set in the configuration
```

Comment:
What do you mean? What constructor?

Reply:
Should I write the sentence like this, "if no database connection instance is passed, it will automatically connect to the default database group, as set in the Database configuration"?

Reply:
Sorry, I don't get what you want to say in the comment.
Where do you pass the database connection instance?
You removed all parameters in the constructor.
https://github.com/codeigniter4/shield/pull/744/files#diff-19dfe8014fc6cf7542ad346ba88dc852a209f343657359347e17149bd8a40006R21

Reply:
Yes, you're right. Initially, I intended to solve the issue using parameters, but in the end, I chose a different approach. I've adjusted the comment accordingly. I hope that's satisfactory.